automatic/clementine/clementine.nuspec

```diff
@@ -1,22 +1,43 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
-<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>clementine</id>
     <version>1.3.1</version>
     <title>Clementine</title>
-    <tags>music mp3 audio media foss cross-platform podcast ogg flac transcode player admin</tags>
+    <tags>clementine music mp3 audio media foss cross-platform podcast ogg flac transcode player admin</tags>
     <owners>chocolatey</owners>
     <authors>David Sansome, John Maguire and Arnaud Bienner</authors>
-    <licenseUrl>https://www.gnu.org/licenses/gpl.html</licenseUrl>
+    <licenseUrl>https://github.com/clementine-player/Clementine/blob/master/COPYING</licenseUrl>
     <projectUrl>http://www.clementine-player.org/</projectUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
-    <copyright />
+    <copyright>Copyright 2010, 2014, John Maguire. Copyright 2014, Krzysztof Sobiecki</copyright>
+    <docsUrl>https://github.com/clementine-player/Clementine/wiki</docsUrl>
+    <mailingListUrl>https://groups.google.com/forum/#!forum/clementine-player</mailingListUrl>
     <description>
 Clementine is a modern music player and library organizer. It is inspired by Amarok 1.4, focusing on a fast and easy-to-use interface for searching and playing your music.
+
+## Features
+- Search and play your local music library.
+- Listen to internet radio from Spotify,Grooveshark, SomaFM, Magnatune, Jamendo, SKY.fm, Digitally Imported, JAZZRADIO.com, Soundcloud, Icecast and Subsonic servers.
+- Search and play songs you've uploaded to Box, Dropbox, Google Drive, and OneDrive
+- Create smart playlists and dynamic playlists.
+- Tabbed playlists, import and export M3U, XSPF, PLS and ASX.
+- CUE sheet support.
+- Play audio CDs.
+- Visualisations from projectM.
+- Lyrics and artist biographies and photos.
+- Transcode music into MP3, Ogg Vorbis, Ogg Speex, FLAC or AAC.
+- Edit tags on MP3 and OGG files, organise your music.
+- Fetch missing tags from MusicBrainz.
+- Discover and download Podcasts.
+- Download missing album cover art from Last.fm and Amazon.
+- Remote control using and Android device, a Wii Remote, MPRIS or the command-line.
+- Copy music to your iPod, iPhone, MTP or mass-storage USB player.
+- Queue manager.
     </description>
     <dependencies>
-      <dependency id="chocolatey-core.extension" version="1.0.4" />
+      <dependency id="chocolatey-core.extension" version="1.0.6" />
```

Comment:
OK, makes sense. I don't think there ever will be a package locking to an exact version or max version of `chocolatey-core.extension` which would lead to conflicts.